services/app/src/pages/game/Game.jsx

```diff
@@ -33,6 +35,17 @@ function Room() {
     );
   };
 
+  const skipMove = (userId) => {
+    if (userId && ws.readyState === wsReadyStates.OPEN) {
+      ws.send(
+        JSON.stringify({
```

Comment:
We should probably move building JSON request payloads to some common file and expose the request payload builder methods to be reused in the components:
```
// app/src/pages/game/Game.jsx

  import { moveSkipped } from 'app/src/api/playerApi.js'
  ...
  const skipMove = userId => {
    if (userId) {
      ws.send(moveSkipped(userId))
    }
  }
```
```
// app/src/api/playerApi.js

import PLAYER_EVENT_TYPES from '...somewhere';

const jsonEventPayload = (eventType, payload) => JSON.stringify({
  type: eventType,
  payload
});

export const moveSkipped = (userId) => 
  jsonEventPayload(
    PLAYER_EVENT_TYPES.MoveSkipped,
    { userId }
  )
```
Same logic should be applied to other events in this file.

Reply:
same here, had similar ideas but was focusing only on task assuming that it will be introduced on refactor task.